Content/Archaeology/BuriedArtifacts/Artifacts.DesertArtifacts.cs

```diff
@@ -0,0 +1,87 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using StarlightRiver.Core;
+using StarlightRiver.Content.Archaeology;
+using StarlightRiver.Content.Items.BuriedArtifacts;
+using StarlightRiver.Helpers;
+using Terraria;
+using Terraria.DataStructures;
+using Terraria.GameInput;
+using Terraria.ID;
+using Terraria.ModLoader;
+using Terraria.ModLoader.IO;
+
+namespace StarlightRiver.Content.Archaeology.BuriedArtifacts
+{
+    public abstract class FossilArtifact : DesertArtifact
```

Comment:
Similar sentiment to the other comment on class creation -- A method to load this content with parameters for basic configuration like spawn chance and size would be much  cleamer